internal/api/operations/create.go

```diff
@@ -2,35 +2,31 @@ package operations
 
 import (
 	"awans.org/aft/internal/db"
+	"fmt"
 )
 
-type UniqueQuery struct {
-	Key string
-	Val interface{}
-}
-
 func (op CreateOperation) Apply(tx db.RWTx) (*db.QueryResult, error) {
 	tx.Insert(op.Record)
 	for _, no := range op.Nested {
-		err := no.ApplyNested(tx, op.Record)
+		err := no.ApplyNested(tx)
 		if err != nil {
 			return nil, err
 		}
 	}
-	out, err := op.Include.One(tx, op.Record.Interface().ID(), op.Record)
+	out, err := op.FindManyArgs.Include.One(tx, op.Record.Interface().ID(), op.Record)
 	if err != nil {
 		return nil, err
 	}
 	tx.Commit()
 	return out, nil
 }
 
-func (op NestedCreateOperation) ApplyNested(tx db.RWTx, parent db.Record) (err error) {
+func (op NestedCreateOperation) ApplyNested(tx db.RWTx) (err error) {
 	tx.Insert(op.Record)
-	tx.Connect(parent.ID(), op.Record.ID(), op.Relationship.ID())
+	tx.Connect(op.Relationship.Source().ID(), op.Record.ID(), op.Relationship.ID())
```

Comment:
This connects the record to the source table itself, not to its parent record. Please add a test case  covering this as well!

Reply:
Are you sure? I got right of passing in the parent for all the nested ops becaue it was cleaner, but if connect and create need them, I can add them into the structs

Reply:
Please write a test and find out :)

Reply:
Will do, but I just added these print statements:

	fmt.Println(op.Relationship.Source().ID())
	fmt.Println(op.Relationship.Target().ID())
	fmt.Println(op.Record.Interface().ID())

and here are the outputs:

887a91b8-3857-4b4d-a633-a6386a4fae25
66783192-4111-4bd8-95dd-e7da460378df
66783192-4111-4bd8-95dd-e7da460378df

pretty sure i'm right?

Reply:
And further confirmed that the parent has the same id as the source

Reply:
If that's true, something has gone seriously awry!